base/tekton.dev/tasks/build-layer.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-layer
+spec:
+  steps:
+    - image: quay.io/podman/stable:latest
+      name: build-layer
+      resources: {}
+      securityContext:
+        privileged: true
+      script: |
+        #!/usr/bin/env bash
+        set -euxo pipefail
+        
+        dnf install -y jq
+        
+        cd /srv/coreos
+
+        OSTREE_VERSION=$(jq -r '."ostree-version"' < builds/latest/$(uname -m)/meta.json)
+        OSTREE_TARFILE=$(jq -r '.images.ostree.path' < builds/latest/$(uname -m)/meta.json)
+
+        # the c9s.repo file is meant to be consumed by a COSA container
+        # where the RPM GPG keys are shipped in /usr/share/distribution-gpg-keys/centos
+        # SCOS however ships the keys in the /etc/pki/rpm-gpg
+        sed -i 's;/usr/share/distribution-gpg-keys/centos;/etc/pki/rpm-gpg;g' src/config/c9s.repo
+        sed -i 's/RPM-GPG-KEY-CentOS-Official/RPM-GPG-KEY-centosofficial/g' src/config/c9s.repo
+
+        cat <<EOF >> src/config/Dockerfile
+        FROM quay.io/okd/centos-stream-coreos-9:4.14-x86_64
+        COPY rpms/ /srv/coreos/rpms/
```

Comment:
It should be `COPY rpms/ /workspace/coreos/rpms/` since this commit https://github.com/okd-project/okd-coreos-pipeline/commit/067ce9d787164487f6a9d5cab770a37b1dd1b99d 
Otherwise we are hitting the issue below :
`error: Updating rpm-md repo 'artifacts': /workspace/coreos/rpms/ was not found` during the step `STEP 5/6: RUN cat /etc/os-release     && rpm-ostree --version     && ostree --version     && rpm-ostree ex rebuild     && rpm-ostree cleanup -m     && rm -rf /var/lib/unbound /srv/coreos/rpms /etc/rpm-ostree/origin.d /etc/yum.repos.d/c9s.repo     && ostree container commit`